internal/store/model/share_token.go

```diff
@@ -0,0 +1,33 @@
+package model
+
+import (
+	"encoding/json"
+
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+)
+
+type ShareToken struct {
+	gorm.Model
+	ID       uuid.UUID `json:"id" gorm:"primaryKey"`
+	Token    string    `json:"token" gorm:"not null;unique"`
+	SourceID uuid.UUID `json:"sourceId" gorm:"not null"`
+	OrgID    string    `json:"orgId" gorm:"not null"`
```

Comment:
why do you need this?

Reply:
You are right, this is not needed, its leftover from development decision changes
I removed it
Thanks